src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+let App = () => {
+    return (
+        <div>Hello World!</div>
+    );
+}
+
+ReactDOM.render(<App />, document.getElementById('app'))
```

Comment:
Also consider installing ESLint to have single code-style within the whole project. I'd recommend to take a look at Airbnb ruleset

Reply:
added ESLint to the project